DataModel/SaveFileToFolder.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace DataModel
+{
+    public class SaveFileToFolder
+    {
+        public static string SaveImage(string fileName)
```

Comment:
Ảnh phải lấy từ client gửi lên dưới dạng byte[] nên bắt buộc nó phải là 
```suggestion
        public static string SaveImage(byte[] content, string originFileName)
```

Reply:
Done